app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,12 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @posts = @user.posts
+  end
 
-  def show; end
+  def show
+    @post = Post.find(params[:id])
+    @user = @post.user
+    @comments = @post.comments
```

Comment:
- [ ] I recognized that when I visit a post page it doesn't display comments on the post, it can just show comments number but we can't see them. Please address that and remember you have to display only 5recent comments
![comments](https://user-images.githubusercontent.com/44745855/185381209-85efe195-eaa1-4200-a2e5-3db89c5712e5.png)